.travis.yml

```diff
@@ -22,6 +22,8 @@ before_install:
   - ulimit -s 1082768
   - curl -L -s https://github.com/golang/dep/releases/download/v0.4.1/dep-linux-amd64 -o $GOPATH/bin/dep
   - chmod +x $GOPATH/bin/dep
```

Comment:
tagential but... @amdprophet can this be removed now?

Reply:
I've left it for now and was planning to see if it can do a check to ensure `Gopkg.toml/lock` are in sync with the vendor directory as part of the CI process. You can remove it for now though.

Reply:
Makes sense 👍